common-lib/src/cmr/common/config.clj

```diff
@@ -240,6 +240,14 @@
   {:default (approved-pipeline-documents)
    :parser #(json/parse-string % true)})
 
+(defconfig generic-ingest-disabled-list
+  "This is a toggle to disallow specified generic concepts from being ingested into CMR,
+   Should be an array of keys with the name of the schema to be disallowed from ingest
+   example [:grid :order-option], would disallow grids and order options from being ingested into CMR
+   if no concepts should be disalllowed set to an empty list"
```

Comment:
Maybe use the terms `prevent` and `prevented` rather than `disallow` and `disallowed` for clarity. Also I'm a bit unclear on `Should be an array`. Is this saying the generic concepts being ingest should be an array of keys?

Reply:
That makes sense! I would probably add in `The config's value` before the `should be an array`.

Reply:
Good point with "disallow" that does evoke that this has something to do with Access control which it does not, will change to 'prevent'